src/timers.cc

```diff
@@ -102,7 +106,8 @@ BENCHMARK_NORETURN static void DiagnoseAndExit(const char* msg) {
 double ProcessCPUUsage() {
 // FIXME We want to use clock_gettime, but its not available in MacOS 10.11. See
 // https://github.com/google/benchmark/pull/292
-#if defined(CLOCK_PROCESS_CPUTIME_ID) && !defined(BENCHMARK_OS_MACOSX)
+#if defined(CLOCK_PROCESS_CPUTIME_ID) && !defined(BENCHMARK_OS_MACOSX) && \
```

Comment:
Please restructure these ifdefs so that the most specialized implementations are at the top, and the generic one is the #else.